lib/modules/datasource/packagist/schema.ts

```diff
@@ -14,14 +14,54 @@ export const ComposerRelease = z.object({
     .transform((x) => x.url)
     .nullable()
     .catch(null),
+  require: z
+    .object({
+      php: z.string(),
+    })
```

Comment:
Based on their docs it sounds like either `require` or `php` could possibly be a string called `__unset`. Will this still gracefully ignore that if found?